spec/spec_helper.rb

```diff
@@ -5,26 +5,69 @@
 
 DEBIAN_OS = %w(Ubuntu Debian)
 REDHAT_OS = %w(RedHat CentOS Fedora Amazon Scientific)
-ALL_OS = DEBIAN_OS + REDHAT_OS
+WINDOWS_OS = %w(Windows)
+
+if RSpec::Support::OS.windows?
```

Comment:
Fantastic stuff with the helper! So much cleaner all across the spec tests. Big thanks for this 🙇 

I guess we didn't find a way to use the params here. It's a shame but no worries.